test/cppsuite/test_harness/workload/database_operation.cxx

```diff
@@ -146,9 +146,10 @@ database_operation::insert_operation(thread_context *tc)
         uint64_t added_count = 0;
         bool committed = true;
         tc->transaction.begin();
+        auto &cursor = ccv[counter].cursor;
```

Comment:
You could store `ccv[counter]` to avoid `ccv[counter].coll.id` below

Reply:
Yeah... I did that.